### ISSUE TITLE
This is the title

### ISSUE THREAD

**Issue by forthrin** [rubocop/rubocop#15189]
### Before submitting

- [x] I have searched the existing issues and this hasn't been reported yet.
- [x] I am running the latest stable version of RuboCop.

### Issue type

False positive

### Cop name

This is the cop

### Steps to reproduce the problem

Create an issue here

### Expected behavior

Title = Title
Cop = Cop

### Actual behavior

Title = Cop
Cop = Title

### RuboCop configuration

```yaml

```

### RuboCop version

```shell
N/A
```

**Comment by forthrin** [comment]
Title and Cop were swapped for #15188. This test came out correct, but do check for issues with the template before closing.
